MAUI/ImageEditor/save.md

```diff
@@ -213,4 +213,108 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+## Reset
+
+The [`Reset`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_Reset) method, resets an image to its original state by discarding the changes performed to it. You can start fresh and explore different creative possibilities.the complete set of changes made in image and also resets the image to original loaded image.
```

Comment:
You can start fresh and explore different creative possibilities. The complete set of changes made in the image and also resets the image to the original loaded image.

Reply:
Updated a content